SPECS/ceph/ceph.spec

```diff
@@ -4,13 +4,15 @@
 
 Summary:        User space components of the Ceph file system
 Name:           ceph
-Version:        16.2.10
-Release:        2%{?dist}
+Version:        18.2.1
```

Comment:
Buddy build seems to be failing.

Reply:
Yep, too many things fetched at build time... trying a different approach now.